cylc/flow/prerequisite.py

```diff
@@ -292,6 +296,11 @@ def get_resolved_dependencies(self):
         E.G: ['1/foo', '2/bar']
 
         """
-        return [f'{point}/{name}' for
-                (point, name, _), satisfied in self.satisfied.items() if
-                satisfied == self.DEP_STATE_SATISFIED]
+        return [
+            f'{point}/{name}' for
+            (point, name, _), satisfied in self.satisfied.items()
+            if (
+                satisfied == self.DEP_STATE_SATISFIED
+                or satisfied.startswith('Artificially satisfied by')
```

Comment:
Potentially fragile, consider extracting "Artificially satisfied" out into a class property as has been done for `DEP_STATE_SATISFIED`

Reply:
Sorry, this was an extension commit that I hadn't intended to push yet.